roles/sft-server/molecule/default/cleanup.yml

```diff
@@ -0,0 +1,27 @@
+# Normally we assume DNS to be created externally-to-ansible.
+#
+# But molecule tests will create their own DNS (see create.yml).
+# Here, we remove the DNS record created in 'create.yml'
+- name: Destroy DNS record
+  hosts: localhost
+  tasks:
+    - name: include molecule variables
+      include_vars: shared_vars.yml
```

Comment:
Molecule seems to know the concept of an inventory. Is there a reason for preferring an *include* over setting variables in an inventory?

Reply:
I will leave this question for @jschaul

Reply:
Molecule auto-manages its inventory, based on the lists of hosts in molecule.yml. It would be possible to use the group_vars functionality by adding something like:

```yaml
platforms:
  - name: instance
    # ....
    groups:
      - group1
provisioner:
  name: ansible
  inventory:
    group_vars:
      group1:
        ...
```

Sounds like a useful refactoring step, 👍🏿

Reply:
See PR https://github.com/wireapp/ansible-sft/pull/17